coins/strat.json

```diff
@@ -0,0 +1,75 @@
+"STRAT": {
+        "meta": {
+            "symbol": "STRAT",
+            "name": "Stratis",
+            "supply": {
+                "max": "xxx"
+            },
+            "web": {
+                "websites": [
+                    "http:\/\/www.steneum.com\/"
+                ],
+                "announcement": null,
+                "source_code": "https:\/\/github.com\/dhabitafx\/steneum",
+                "explorers": [
+                    "http:\/\/stratisplatform.com"
+                ],
+                "coinmarketcap": "https:\/\/coinmarketcap.com\/currencies\/stratis\/",
+                "message_board": [],
+                "chat": [
+                    "xxxx"
+                ]
+            },
+            "original_codebase": "bitcoin"
+        },
+        "main": {
+            "symbol": "STRAT",
+            "name": "Stratis - Mainnet",
+            "testnet": false,
+            "port": 26965,
+            "port_rpc": 26966,
+            "protocol": {
+                "magic": "0xe626170d"
+            },
+            "seeds_dns": [],
+            "prefixes": {
+                "private": 191,
+                "public": 63,
+                "scripthash": 125,
+                "bip32": {
```

Comment:
["bip32"] is old format.  latest code uses ["extended"]["xpub"].  Please  ensure you are up to date with master.